app/models/nc-news.model.js

```diff
@@ -1,8 +1,16 @@
 const db = require('../../db/connection')
+const fs = require('fs/promises')
 
 exports.selectTopics = () => {
     return db.query('SELECT * FROM topics')
     .then(({rows}) => {
         return rows;
     })
+}
+
+exports.selectEndpoints = () => {
+    return fs.readFile(`${__dirname}/../../endpoints.json`, 'utf-8').then((endpoints)=> {
+        const endpointFileData = JSON.parse(endpoints)
+        return endpointFileData
+    })
```

Comment:
To slightly streamline this logic, you can require in the endpoints.json directly in the same way you did in your test.

Also - you don't actually need a model, all this logic can live in the controller. This is because it doesn't actually interact with the database and the endpoints.json is readily available